internal/schema/0.12/resource_block.go

```diff
@@ -42,7 +42,8 @@ func resourceBlockSchema(v *version.Version) *schema.BlockSchema {
 			"outside of the scope of a module."),
 		Body: &schema.BodySchema{
 			Extensions: &schema.BodyExtensions{
-				Count: true,
+				Count:   true,
+				ForEach: true,
```

Comment:
```suggestion
				ForEach: true, // for_each was introduced in 0.12.6, but for simplicity we report it for all 0.12+
```
Just a note to avoid any confusion of future readers.